pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -425,7 +425,7 @@ func (a *Bootstrap) addParentFiles(dependencies asset.Parents) {
 
 	journal := &tls.JournalCertKey{}
 	dependencies.Get(journal)
-	a.Config.Storage.Files = append(a.Config.Storage.Files, ignition.FilesFromAsset(rootDir, "systemd-journal-gateway", 0600, journal)...)
+	a.Config.Storage.Files = append(a.Config.Storage.Files, ignition.FilesFromAsset(rootDir, "root", 0600, journal)...)
```

Comment:
we can now move this to https://github.com/openshift/installer/pull/1445/commits/5c9b4e82b8518c64309148c19a45c6681bd75524#diff-2d9b655c8052ca2e3bdd9004778c75bcR416

Reply:
Fixed.